### PR TITLE
[Windows] Add SQL OLEDB Driver to win19 and win22

### PR DESCRIPTION
**Pull request by shilovmaksim**
# Description
Add SQL OLEDB Driver to win19 and win22

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-SQLOLEDBDriver.ps1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+################################################################################
+##  File:  Install-SQLOLEDBDriver.ps1
+##  Desc:  Install SQL OLEDB Driver
+################################################################################
+
+$SQLOLEDBDriverName = "msoledbsql.msi"
+$SQLOLEDBDriverUrl = "https://go.microsoft.com/fwlink/?linkid=2183083"
+Install-Binary -Url $SQLOLEDBDriverUrl -Name $SQLOLEDBDriverName -ArgumentList ("ADDLOCAL=ALL", "IACCEPTMSOLEDBSQLLICENSETERMS=YES", "/qn")
```
[the remark this answers is not in the document]
Comment (shilovmaksim) [reply images/win/scripts/Installers/Install-SQLOLEDBDriver.ps1:8]: I'm not sure. I used the same approach as in [Install-SQLPowerShellTools.ps1](https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Installers/Install-SQLPowerShellTools.ps1).
Could you please advise here?

**Comment by freddy123098** [comment]
@al-cheb can u run the failed ones by using /azp, command

File: images/win/scripts/Tests/Tools.Tests.ps1 [hunk 2]
```diff
@@ -201,3 +201,9 @@ Describe "Kotlin" {
         "$toolName -version" | Should -ReturnZeroExitCode
     }
 }
+
+Describe "SQL OLEDB Driver" {
+    It "SQL OLEDB Driver" {
+        "HKLM:\SOFTWARE\Microsoft\MSOLEDBSQL'" | Should -Exist
```
Comment (al-cheb) [inline images/win/scripts/Tests/Tools.Tests.ps1:207]: Remove ' quote

File: images/win/scripts/Installers/Install-SQLOLEDBDriver.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (al-cheb) [reply images/win/scripts/Installers/Install-SQLOLEDBDriver.ps1:8]: Ok, let's skip.

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 3]
```diff
@@ -223,6 +223,11 @@ $markdown += New-MDList -Style Unordered -Lines (@(
     ) | Sort-Object
 )
 
+if (!(Test-IsWin16))
+{
+    $markdown += (Get-SQLOLEDBDriverVersion)
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:228]: remove ()

Comment (shilovmaksim) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Done

File: images/win/scripts/Tests/Tools.Tests.ps1 (hunk 2 above)
Comment (shilovmaksim) [reply images/win/scripts/Tests/Tools.Tests.ps1]: Done

**Review by al-cheb — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by miketimofeev — CHANGES_REQUESTED** [review]
Something went wrong
image

**Comment by shilovmaksim** [comment]
> Something went wrong image

image

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**